app/src/main/java/chat/rocket/android/util/extensions/TextView.kt

```diff
@@ -0,0 +1,13 @@
+package chat.rocket.android.util.extensions
+
+import android.content.Context
+import android.os.Build
+import android.widget.TextView
+
+fun TextView.setTextViewAppearance(context: Context, style: Int) {
```

Comment:
🛑 It's better to add this extension function on Widget.kt instead of creating a TextView.kt.